proposals/collection-expression-arguments.md

```diff
@@ -278,7 +202,7 @@ If *collection_arguments* is included and is not the first element in the collec
 
 If the *argument list* contains any values with *dynamic* type, a compile-time error is reported ([LDM-2025-01-22](https://github.com/dotnet/csharplang/blob/main/meetings/2025/LDM-2025-01-22.md#conclusion-1)).
 
-#### Constructors
+### Constructors
```

Comment:
nit: Each section is meant to be about a category of target types. This section is not so much about "constructors" as it is about types for which we decide to use the constructor.
I don't know how the four big scenarios/categories of types where labelled in collection expression spec, but I'd try to use consistent terminology if we have a workable one.

Same comment applies to the "CollectionBuilderAttribute methods" section.